src/Components/Header.tsx

```diff
@@ -0,0 +1,26 @@
+import { NavLink } from 'react-router-dom';
+
+export const Header: React.FC = () => {
+  return (
+    <>
+      <nav className="navbar is-fixed-top has-background-light" data-cy="nav">
+        <div className="navbar-menu">
+          <div className="navbar-start">
+            <NavLink
+              to="/"
+              className={({ isActive }) => `navbar-item${isActive ? ' is-active' : ''}`}
```

Comment:
using ternary with an empty string is a weak solution

Reply:
also, this callback looks a new reusable one

Reply:
Agree, fixed